src/core/core-impl.hpp

```diff
@@ -58,9 +58,14 @@ class compositor_core_impl_t : public compositor_core_t
 
     wlr_seat *get_current_seat() override;
     uint32_t get_keyboard_modifiers() override;
+    void warp_cursor(wf::pointf_t pos) override;
+    /**
+     * The functions to hide and set the cursor use reference counting.
+     * A call to hide must come before a matched call to set.
```

Comment:
this doesn't make sense to me, please add `unhide_cursor` or rename to `set_cursor_visibility(bool is_visible)` which increments/decrements the counter

Reply:
I considered unhide_cursor() but I didn't know what cursor it should show when unhiding. Should it take a string as arg? Assume "default"? Show the last cursor that was displayed?

Reply:
I'd display 'default'